frontend/src/pages/auth/components/sign-up-form/sign-up-form.tsx

```diff
@@ -12,44 +14,53 @@ type Properties = {
 };
 
 const SignUpForm: React.FC<Properties> = ({ onSubmit }: Properties) => {
-  const { control, errors, handleSubmit } = useAppForm<UserSignUpRequestDto>({
-    defaultValues: DEFAULT_SIGN_UP_PAYLOAD,
-    validationSchema: userSignUpValidationSchema,
-  });
-
-  const handleFormSubmit = useCallback(
-    (event_: React.BaseSyntheticEvent): void => {
-      void handleSubmit(onSubmit)(event_);
-    },
-    [handleSubmit, onSubmit],
-  );
+  const nameField: FormField<UserSignUpRequestDto> = {
```

Comment:
1) There is no need to name objects, just create an array called `formFields`  
2) Move the field definition out of the component.